### PR TITLE
Canaan Davis

### PR DESCRIPTION
**Pull request by canaandavis**

### CODE REVIEW

File: diagnostic.rb [hunk 1]
```diff
@@ -4,6 +4,9 @@
 # # # # # # #
 
 # TODO: Write a method caled `toggle_oven`
+def toggle_oven(value)
+  value ? "The oven is now on" : "The oven is now off"
```
Comment (truffaut) [inline diagnostic.rb:8]: Nice use of the ternary operator! Do you know the other ways to write this method as well?

File: diagnostic.rb [hunk 2]
```diff
@@ -160,6 +172,8 @@ class HashProblems
 
   def self.create_empty_hash
     # TODO: Complete this method
+    # Hash.new
+    {}
```
Comment (truffaut) [inline diagnostic.rb:176]: nice, looks like you know both ways of doing this :thumbsup:

**Comment by truffaut** [comment]
Great work!

File: diagnostic.rb (hunk 1 above)
Comment (canaandavis) [inline diagnostic.rb:8]: Lets see...

def toggle_oven(value)
  if value == true
    return "The oven is now on"
  else
    return "The oven is now off"
  end
end

# I don't really like this one, though, I think it should work

def toggle_oven(value)
  return "The oven is now on" if value == true
  "The oven is now off"
end

**Pull request closed without merging**